### DISCUSSION TITLE
Add to the guides "Real" Manual Install of Tailwind

### DISCUSSION THREAD

**Discussion by MrAmericanMike** [skeletonlabs/skeleton#2732] [General]
On the guide the options are using Skeleton CLI or "Manual install" but the manual install is really using svelte-add CLI which assumes a lot of conditions.

Would be nice to have real "Manual" instructions of how to add Tailwind to a SvelteKit project.

Following svelte-add causes this in my project.

![image](https://github.com/skeletonlabs/skeleton/assets/8037559/9d7431fa-78bb-47cd-b0c2-3549358da9c8)

I guess I will find this on Tailwind documentation, but having it somewhere on Skeleton guides would help too.

**Comment by endigo9740** [comment]
@MrAmericanMike Keep in mind we've provided support for Skeleton for nearly 2 years now. While I agree with your sentiment, the more steps involved, the more friction folks have had in the past. People will miss a step or make incorrect assumptions, and then that's more support issues for us to handle. Our goal is to lead folks into a "pit of success", and so far this has been one of the best way of achieving that, with extremely positive results.

We'll probably keep Svelte-Add as the default going forward, but I could see us linking to the instructions for folks that wish to handle this themselves. There's nothing stopping folks from doing that today though.

**Reply by MrAmericanMike** [reply]
Thanks. Yes I much rather have a manual steps so I am aware of what modifications are expected instead of having svelte-add taking guesses for us and maybe failing without us knowing.

```
People will miss a step or make incorrect assumptions
```

I agree with that, but I much rather have the steps, even if it's more involved it's the only way to learn all the things that are needed for a package to work, not only that, but also we learn that way what goes where and what we should change in case our SvelteKit setup is different from the default (Which in my case it is, not much different, but it is different)
